Features/foutafhandeling.feature

```diff
@@ -45,207 +45,633 @@ Bij een fout op een parameter krijgt in "invalid-params" attribuut "code" een va
+      | panden                | ?point=abc,def                                                            | point                | number    | Waarde is geen geldig decimaal getal.                      |
+      | adresseerbareobjecten | ?bbox=135207,457400,135418,457162&geconstateerd=Nee                       | geconstateerd        | boolean   | Waarde is geen geldige boolean.                            |
+      | adresseerbareobjecten | ?bbox=135207,457400,135418,457162&oppervlakte[min]=-1&oppervlakte[max]=1  | oppervlakte[min]     | minimum   | Waarde is lager dan minimum 1.                             |
+      | panden                | ?bbox=135207,457400,135418,457162&bouwjaar[min]=2000&bouwjaar[max]=20200  | bouwjaar[max]        | maximum   | Waarde is hoger dan maximum 9999.                          |
+      | panden                | ?point=98095.02                                                           | point                | minItems  | Array bevat minder dan 2 items.                            |
+      | panden                | ?point=98095.02,438495.09,0                                               | point                | maxItems  | Array bevat meer dan 2 items.                              |
+      | adressen              | ?pandIdentificatie=0345100002016017X                                      | pandIdentificatie    | pattern   | Waarde voldoet niet aan patroon ^[0-9]{16}$.               |
+      | adresseerbareobjecten | ?bbox=135207,457400,135418,457162&type=fout                               | type                 | enum      | Waarde heeft geen geldige waarde uit de enumeratie.        |
+
+  Abstract scenario: meerdere parameter waarden voldoen niet aan gespecificeerde type
+    Als <path><query string> wordt aangeroepen
+    Dan bevat de response ten minste de volgende velden
+    | naam     | waarde                                        |
+    | title    | Een of meerdere parameters zijn niet correct. |
+    | status   | 400                                           |
+    | instance | <path><query string>                          |
+    | code     | paramsValidation                              |
+    En bevat de response de invalidParams alleen voor de eerste foutieve parameter waarde
+    | name        | code   | reason   |
+    | <parameter> | <code> | <reason> |
+
+    Voorbeelden:
+    | path              | query string                                                                                                            | parameter     | code             | reason                                              |
+    | verblijfsobjecten | ?pandIdentificatie=0345100002016017X&geldigOp=23-04-2019                                                                | identificatie | integer          | Waarde voldoet niet aan patroon ^[0-9]{16}$.        |
```

Comment:
De waarde `23-04-2019` is geen geldige waarde voor geldigOp, dat had `2019-04-23` moeten zijn? Als je dat corrigeert krijg je een validatiemelding waarbij in deze tabel de waarde voor parameter `pandIdentificatie` wordt, code `pattern`, en de reason `Waarde voldoet niet aan patroon ^[0-9]{4}10[0-9]{10}$.`

Reply:
De waarde 23-04-2019 is hier bewust fout gekozen, omdat als er merdere type fouten zijn er alleen een fout op de eerste optreedt.